recipes/date/all/patches/cmake.patch

```diff
@@ -12,15 +12,26 @@ index f025a3a..072c644 100644
  option( USE_SYSTEM_TZ_DB "Use the operating system's timezone database" OFF )
  option( USE_TZ_DB_IN_DOT "Save the timezone database in the current folder" OFF )
  option( BUILD_SHARED_LIBS  "Build a shared version of library" OFF )
-@@ -61,6 +58,11 @@ else( )
- 	set( OPTIONAL_LIBRARIES ${CURL_LIBRARIES} )
- endif( )
- 
+@@ -56,16 +53,18 @@ else( )
+ 	target_compile_definitions( tz PRIVATE -DUSE_AUTOLOAD=1 )
+ 	target_compile_definitions( tz PRIVATE -DHAS_REMOTE_API=1 )
+ 	target_compile_definitions( tz PUBLIC -DUSE_OS_TZDB=0 )
+-	find_package( CURL REQUIRED )
+-	include_directories( SYSTEM ${CURL_INCLUDE_DIRS} )
+-	set( OPTIONAL_LIBRARIES ${CURL_LIBRARIES} )
++endif( )
++
 +if( BUILD_SHARED_LIBS )
 +	target_compile_definitions( tz PRIVATE -DDATE_BUILD_DLL=1 )
```

Comment:
The `-D` is removed in cmake since it not cross platform

Reply:
Hm, slightly confused - can you please elaborate?
I am looking at cmake docs: https://cmake.org/cmake/help/v3.12/command/target_compile_definitions.html
And it seems these are equivalent. 

I can remove the -D but that would add more changes to the cmake.patch which probably we should be keeping to minimimum.

Reply:
You are correct, it's just a styling comment.

I am suggesting using the modern cmake approach but, it wont make any difference

Reply:
Uf - nice! :) Thanks for getting back to me, relieved! resolving.